packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/download-sample.md

```diff
@@ -0,0 +1,31 @@
+### Create a React app (optional)
+
+If you don't have an existing React app, you can quickly create an app by using the [Create React App command](https://create-react-app.dev/docs/getting-started/):
+
+```js
+npx create-react-app okta-app
+cd okta-app
+```
+
+### Install dependencies
+
+Add the [latest version of the Okta Sign-In Widget](https://github.com/okta/okta-signin-widget/releases) library to your React app. You can install it by using `npm` from your root app directory:
+
+```bash
+npm install @okta/okta-signin-widget@latest
```

Comment:
I would hard-code this to the latest release because sometimes `latest` is broken and we want to QA a snapshot in time. Ideally `latest` always works, but I've had issues with it myself.